src/Symfony/Component/Translation/MessageCatalogue.php

```diff
@@ -32,6 +32,10 @@ class MessageCatalogue implements MessageCatalogueInterface, MetadataAwareInterf
      */
     public function __construct(?string $locale, array $messages = [])
     {
+        if (null === $locale) {
+            @trigger_error(sprintf('Passing "null" to "$locale" in %s has been deprecated since Symfony 4.4 and will throw in 5.0.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
```suggestion
            @trigger_error(sprintf('Passing "null" to "$locale" in %s has been deprecated since Symfony 4.4 and will throw an exception in 5.0.', __METHOD__), E_USER_DEPRECATED);
```

Or an error?

Reply:
> Or an error?

It will be a `TypeError`, actually.